UnitystationLauncher/ViewModels/PreferencesPanelViewModel.cs

```diff
@@ -160,6 +175,26 @@ await MessageBoxBuilder.CreateMessageBox(MessageBoxButtons.Ok, "Error moving ins
                 await MessageBoxBuilder.CreateMessageBox(MessageBoxButtons.Ok, "Invalid installation path", invalidReason).ShowAsync();
             }
         }
+        
+        public async Task OnAllowCodeScanChangedAsync()
+        {
+            if (EnableCodeScan == false)
+            {
+                IMsBox<string> msgBox = MessageBoxBuilder.CreateMessageBox(
+                    MessageBoxButtons.YesNo,
+                    "Warning",
+                    "For security reasons, we recommend you never disable the codescan feature unless you're trying to play on older servers that are no longer supported.\n Are you sure you want to proceed?"
```

Comment:
even if you are trying to play on older servers, this should not be recommended. I think the text should be

> For security reasons, we recommend you never disable the codescan feature unless you understand what you are doing and you trust the server you are trying to play on.\n Are you sure you want to proceed?

Reply:
updated